src/backends/GradientBackends/ElementReverseDiffGradient.jl

```diff
@@ -0,0 +1,67 @@
+export ElementReverseDiffGradient
+
+_ni(element_variables::Vector{Int}) =
+  isempty(element_variables) ? 0 : maximum(element_variables)
+
+"""
+    element_gradient_tape = compiled_grad_element_function(element_function::T; ni::Int = length(ExpressionTreeForge.get_elemental_variables(element_function)), type = Float64) where {T}
+
+Return the `elment_gradient_tape::GradientTape` which speed up the gradient computation of `element_function` with `ReverseDiff`.
+"""
+function compiled_grad_element_function(
+  element_function::G;
+  element_variables::Vector{Int} = ExpressionTreeForge.get_elemental_variables(element_function),
+  ni::Int = _ni(element_variables),
+  type = Float64,
+) where {G}
```

Comment:
```suggestion
  type::Type{T} = Float64,
) where {G, T}
```
and below you can use `T` instead of `type`. This will help Julia infer the types.

Reply:
And I sould correct `Float64` as well by `Type{Float64}`?

Reply:
Nope, you can leave as suggested

Reply:
Hi think you missed this one?